index.html

```diff
@@ -12,10 +12,76 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="logo">
+
+    <img src="img/karma-logo.svg" alt="logo">
+ 
+    </header>
+    <!-- HAMBURGER -->
+    <div class="hamburger">
+        <img src="img/menu-hamburger.svg" alt="hamburger">
+
+    </div>
+ 
+    
+<nav class="navbar">
+        <ul>  
+            <li> <a href="#login"class="navbar">Login</a></li>
+            <li> <a href="#help" class="navbar">Help</a></li>
+            <li> <a href="#help" class="navbar">Blog</a></li>
+            <li> <a href="#store"class="navbar">Store</a></li>
+            <li> <a href="#how it works"class="navbar">How It Works</a></li>
+            <li> <b><a href="#meet karma"class="navbar" >Meet Karma</a> </b></li>
+      
+        </ul>
+    
+    </nav>
+<main>
+    <section class="photo"> 
+            <h3>Introducing Karma</h3>
+            <h4>Bring WiFi with you,everywhere you go.</h4>
+           
+
+    </section>
+    <button class="learn"
+        type="button">
+    Learn More
+</button>
+    
+    <section class="icon">
+       <p>Everyone needs a little Karma.</p>
+        <img src="img/icon-devices.svg" alt="device">
+        <img src="img/icon-coffee.svg" alt="coffee">
+        <img src="img/icon-refill.svg" alt="refill">
+        
+    </section>
+    <section class="specs">
+        <p>Internet for all devices</p>
+        <p>Boost your productivity</p>
+        <p>Pay as You Go</p>
+         
```

Comment:
Move these descriptions so that they are with their respective image. For example:
```
       <div class="icon-wrapper">
           <img src="img/icon-devices.svg" alt="device">
           <p>Internet for all devices</p>
       </div>
``` 
(for each icon)